docs/getting-started/clients/mobile/watchos/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# watchOS app
+
+## Requirements
+
+Follow the [iOS Setup](../ios/index.mdx).
+
+In order for everything to work properly **devices** are needed. On simulators the synchronization
+won't work (and some other parts may not work as well). Also, have Bluetooth enabled if possible to
+ease the sync between devices and the debugging communication.
+
+It's recommended to read the [Watch Architecture](../../../../architecture/mobile-clients/watchOS)
+as well.
+
+## MacOS Setup
+
+Having followed the macOS setup of iOS, no additional configuration is needed given that when the
+project is opened in XCode it will automatically have the provisioning profiles set up.
+
+## Debugging
+
+There are two parts from where to debug:
+
+- From Visual Studio for Mac (the iOS app)
+- From XCode (the watchOS app)
+
+For now, there is no way to debug both apps (iOS and watchOS) at the same time given that from
+Xamarin there is no access to debug information of the watchOS app and from XCode an iOS stub app is
+installed on the iPhone to debug it. So, at the moment of debugging one needs to choose which part
+to have information about, therefore whether to debug from VS4M or from XCode.
+
+:::caution
+
+When debugging from XCode the Xamarin iOS app will be replaced with the stub one from XCode. So any
+configuration on the iOS app will be lost (like server urls)
+
+When debugging from VS4M, uninstall the previous watchOS app (if any) from the Apple Watch in
+between builds to have it always up to date (there are times that if one doesn't uninstall the
+previous watchOS app it doesn't get updated)
+
+:::
+
+### Building
+
+Given that the Xamarin iOS app needs the output of the build of Xcode, one needs to build the
+watchOS app from XCode first and then from VS4M build the iOS app to run it on the device.
+
+The output of XCode build is stored in a location pretty similar to the next one that is configured
```

Comment:
> pretty similar to the **next** one 

was the "next" intended?

Reply:
Yes, to indicate the "next" part or line